target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -259,6 +259,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:1" "3:lan:3" "4:lan:2"
 		;;
+	ubnt,edgeswitch-5xp)
+		ucidef_set_interface_wan "eth1"
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4" "5:lan:5"
```

Comment:
If internal and external are the same, you do not have to add the number a second time:
`"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan" "5:lan"`

Reply:
Fixed, thanks!